sql/voting.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE IF EXISTS survey_responses;
+DROP TABLE IF EXISTS survey_question_choices;
+DROP TABLE IF EXISTS survey_questions;
+DROP TABLE IF EXISTS survey_question_types;
+DROP TABLE IF EXISTS surveys;
+
+CREATE TABLE surveys (
+    survey_id      INT           NOT NULL AUTO_INCREMENT,
+    title          VARCHAR(100)  NOT NULL,
+    description    TEXT,
+    start_time     DATETIME      NOT NULL,
+    end_time       DATETIME      NOT NULL,
+    access_key     CHAR(64)      NOT NULL, -- hex string
+    group_id       INT,                    -- restrict to this group if non-NULL
+    auth           BOOLEAN       NOT NULL, -- require authentication (UID and birthday)
+    public         BOOLEAN       NOT NULL, -- whether survey should appear in the list of active surveys
+    results_shown  BOOLEAN       DEFAULT FALSE, -- whether survey should appear in the list of results
+    creator        INT           NOT NULL, -- user id of survey creator
+
+    PRIMARY KEY(survey_id),
+    FOREIGN KEY(group_id) REFERENCES groups(group_id) ON DELETE SET NULL,
+    FOREIGN KEY(creator) REFERENCES members(user_id) ON DELETE CASCADE
```

Comment:
don't want this. if a user decides to delete their account or something, we will want to keep the surveys they started as well.